docs/source/recipes/Non-streaming-ASR/librispeech/zipformer_ctc_blankskip.rst

```diff
@@ -0,0 +1,453 @@
+Zipformer CTC Blank Skip
+========================
+
+.. hint::
+
+   Please scroll down to the bottom of this page to find download links
+   for pretrained models if you don't want to train a model from scratch.
+
+
+This tutorial shows you how to train a Zipformer model based on the guidance from 
```

Comment:
What about "a Zipformer transducer model"? People might be confused about whether this model is a transducer or a CTC.

Reply:
OK